src/mscorlib/corefx/System/Threading/ClrThreadPoolBoundHandle.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Threading
     ///     Represents an I/O handle that is bound to the system thread pool and enables low-level
     ///     components to receive notifications for asynchronous I/O operations.
     /// </summary>
-    public sealed partial class ThreadPoolBoundHandle : IDisposable
```

Comment:
This should stay as is - this type was moved from corefx, but the redundant copy in corefx was not removed yet. I have pending change to clean it up.